src/validate-and-gen-trees.sh

```diff
@@ -4,7 +4,8 @@
 # Does the user have all the IETF published models.
 #
 if [ ! -d ../../yang-parameters ]; then
-   rsync -avz --delete rsync.iana.org::assignments/yang-parameters ../../
+    rsync -avz --delete rsync.iana.org::assignments/yang-parameters ../../
+    rm -rf ../../yang-parameters/*bfd*
```

Comment:
This will collide with my issue.  Did you want me to hold off committing that?

Reply:
I will resolve this before merging.